schema/SchD.json

```diff
@@ -0,0 +1,472 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/fecgov/fecfile-validate/blob/main/schema/SchD.json",
+    "version": "8.3.0.1",
+    "title": "FEC Sch D",
+    "description": "SCHEDULE D - DEBTS AND OBLIGATIONS  (Itemized for each one)",
+    "type": "object",
+    "required": [
+        "form_type",
+        "filer_committee_id_number",
+        "transaction_id",
+        "creditor_organization_name",
+        "creditor_last_name",
+        "creditor_first_name"
+    ],
+    "fec_recommended": [
+        "creditor_street_1",
+        "creditor_city",
+        "creditor_state",
+        "creditor_zip",
+        "purpose_of_debt_or_obligation",
+        "beginning_balance_this_period",
+        "incurred_amount_this_period",
+        "payment_amount_this_period",
+        "balance_at_close_this_period"
+    ],
+    "properties": {
+        "form_type": {
+            "title": "FORM TYPE",
+            "description": "",
+            "type": "string",
+            "minLength": 0,
+            "maxLength": 8,
+            "pattern": "^[ -~]{0,8}$",
+            "examples": [
+                "SD10"
+            ],
+            "fec_spec": {
+                "COL_SEQ": 1,
+                "FIELD_DESCRIPTION": "FORM TYPE",
+                "TYPE": "A/N-8",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "SD10",
+                "VALUE_REFERENCE": "SD[line# ref]",
+                "RULE_REFERENCE": "Appendix C",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "filer_committee_id_number": {
+            "title": "FILER COMMITTEE ID NUMBER",
+            "description": "",
+            "type": "string",
+            "minLength": 0,
+            "maxLength": 9,
+            "pattern": "^[ -~]{0,9}$",
+            "examples": [
+                "C00123456"
+            ],
+            "fec_spec": {
+                "COL_SEQ": 2,
+                "FIELD_DESCRIPTION": "FILER COMMITTEE ID NUMBER",
+                "TYPE": "A/N-9",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "C00123456",
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": "this is the ID of the Committee Account the report/transaction is associated with",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "transaction_id": {
+            "title": "TRANSACTION ID NUMBER",
```

Comment:
might as well change this to "TRANSACTION ID" since we've taken that liberty with the other schedules

Reply:
Ah, this has bothered me in the past - great!